src/components/ReviewPage.jsx

```diff
@@ -0,0 +1,13 @@
+import { useParams } from "react-router-dom"
```

Comment:
Not too sure this needs to be a separate component, as you can bring this logic into SingleReview and just have the id from useParams() being grabbed inside the component. Unless you're doing it for styling reasons or future functionality...